fastlane/Fastfile

```diff
@@ -458,15 +469,61 @@ platform :ios do
   #####################################################################################
   desc 'Downloads localized metadata for App Store Connect from GlotPress'
   lane :download_localized_strings_and_metadata do |options|
-    # WordPress: leverage the default release toolkit setup.
-    # It's not clear in the action name, but this downloads both `.strings`
-    # translations for the app and App Store Connect metadata.
-    ios_update_metadata(options)
-    # Jetpack: use the custom setup. Notice that we don't need extra steps to
-    # download `.strings` because Jetpack uses the same codebase as WordPress.
+    # Download `Localizable.strings` translations used within the app
+    parent_dir_for_lprojs = File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources')
+    ios_download_strings_files_from_glotpress(
+      project_url: 'https://translate.wordpress.org/projects/apps/ios/dev/', # TODO: Use a constant to DRY stuff
+      # TODO: Fill this in
+      locales: { gp1: lproj1, gp2: lproj2, … }, # TODO: Provide a big constant for this list somewhere. (See `ALL_LANGS` from `update-translation.rb`)
+      download_dir: parent_dir_for_lprojs
+    )
+
+    # Then redispatch the appropriate subset of translations back to the `InfoPlist.strings` and `Sites.strings` files in corresponding `*.lproj` dirs
+    parent_dir_for_intents_target_lprojs = File.join('WordPress', 'WordPressIntents')
+    ios_extract_keys_from_strings_files(
+      source_parent_dir: parent_dir_for_lprojs
+      target_original_files: [
+        File.join(parent_dir_for_lprojs, 'en.lproj', 'InfoPlist.strings'),
+        File.join(parent_dir_for_intents_target_lprojs, 'en.lproj', 'Sites.strings'),
+      ]
+    )
+
+    # Finally, also download the AppStore metadata (app title, keywords, etc.)
+    # @FIXME: Replace this whole lane with a call to the future replacement of `gp_downloadmetadata` once it's implemented in the release-toolkit (see paaHJt-31O-p2).
```

Comment:
This FIXME might be for another PR (once `gp_downloadmetadata`'s successor is ready)

Reply:
_Braindump on `default` vs `en-US`_

Note that while we're at it (and/or also while we'll be working on updating the keywords soon, see pdjUbD-8v-p2), we might want to take the occasion to revisit the distinction of what goes in `en-US` vs `default` folders.

See also [the docs for `deliver`](https://docs.fastlane.tools/actions/deliver/#available-metadata-folder-options) about this. Typically:
 - `en-US` is the source locale, that contains what we want to send to GlotPress for translation to other locales
 - `default` is used by fastlane for when a given locale folder does not have a given `.txt` file for a field

Some thoughts on our options:
 - Making `default` our source of truth (in which we'd edit the `release_notes.txt` and `keywords.txt` files to put the originals in) could be a nice idea.
 - On the other hand, we might not absolutely want _every_ metadata field to be sent to GlotPress for translation (e.g. `support_url ` or `marketing_url`), so having those not-to-be-translated values in `default`, while having the originals we want to send for translation in `en-US` could make more sense and be more flexible
 - Even though sending even `support_url` and the likes to GlotPress… might not be that big of a deal after all: the translations would currently just be the same as the original, but that would allow us in the future to provide URLs to localized content (e.g. `https://wordpress.com/fr/support`…). After all, those fields are localizable in ASC (which is why `deliver` consider them localizable too), so…

That might also give us the opportunity to use `default` vs `en-US` as a way to differentiate which `.txt` files should go into the `.po` and to GlotPress for translation (`en-US`) and which `.txt` files we don't want to translate (e.g. `keywords.txt` could be moved to `default` during an ASO experiment to stop being included into the `.po`, and moved back into `en-US` once we're ready to translate them again…)

Reply:
> I like the idea of having `default` being also the source of truth for GlotPress.

At that point, we could handle the `en-US` folder by leaving it empty and adding a `.gitkeep` file, so we:

- Wouldn't need to copy files or use `en-GB` as the source for `en-US` (which, by the way, could lead to inappropriate copy, e.g. "colour" vs "color"); and
- Would let `deliver` fallback to the `default` content, making the things full circle: The copy used in ASC for `en-US` is the one from which all (excluding experiments) the other languages have been derived.